app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cm, first_file, second_file = command.split()
```

Comment:
you should add a check that `command.split()` is equal to `3`, only after that perform all actions


You code will not work if command will be one of:
"mv"
"mv file1.txt"
"mv file1.txt file2.txt file3.txt"